tests/test_conditions.py

```diff
@@ -22,3 +27,69 @@ def test_get_parametric_overrides():
     condition_df.loc[0, 'fixedParameter1'] = 'parameterId'
 
     assert conditions.get_parametric_overrides(condition_df) == ['parameterId']
+
+
+def test_get_condition_df():
+    """Test conditions.get_condition_df."""
+    # condition df missing ids
+    condition_df = pd.DataFrame(data={
+        CONDITION_NAME: ['Condition 1', 'Condition 2'],
+        'fixedParameter1': [1.0, 2.0]
+    })
+
+    with tempfile.NamedTemporaryFile(mode='w', delete=False) as fh:
+        file_name = fh.name
+        condition_df.to_csv(fh, sep='\t', index=False)
```

Comment:
use `write_condition_df`? makes it more future-proof... 
same for other tables and respective functions. some would requires an index.
not too passionate about it.

Reply:
there is a separate test for writing and reading below. for these tests here I wanted to be sure what is written to file.